runhouse/servers/cluster_servlet.py

```diff
@@ -118,6 +119,22 @@ def update_autostop(self):
     async def aget_cluster_config(self) -> Dict[str, Any]:
         return self.cluster_config
 
+    async def aupdate_status_check_interval_in_cluster_config(self):
+        cluster_path = Path("~/.rh/cluster_config.json").expanduser()
+        with open(cluster_path) as cluster_local_config:
+            local_config = json.load(cluster_local_config)
+            local_interval = local_config.get("status_check_interval")
+            servlet_interval = self.cluster_config.get("status_check_interval")
+            if local_interval != servlet_interval:
```

Comment:
this relates to my other comment, but this doesn't account for None values which it probably should